app/views/visitor_mailer/booked.html.erb

```diff
@@ -32,6 +32,20 @@
   <p><%= t(".this_is_a_closed_visit_html") %></p>
 <% end %>
 
+<% if @visit.visitors.any?(&:banned?) %>
+  <% @visit.visitors.select(&:banned?).each do |v| %>
+    <p><%= t('.visitor_banned', name: v.anonymized_name) %></p>
```

Comment:
I've tested locally and the people that are banned or not on the list are easy to miss in the email. Using the same formatting as the approved visitors could be a solution

<img width="594" alt="screen shot 2016-02-25 at 09 54 22" src="https://cloud.githubusercontent.com/assets/6356/13316104/462bec10-dba6-11e5-829f-b6d4470ac042.png">